Gemfile

```diff
@@ -1,34 +1,40 @@
 source 'https://rubygems.org'
 
 group :tools do
-  gem 'rubocop'
   gem 'guard', '<= 2.12.0'
   gem 'guard-minitest', '~> 2.4.4'
   gem 'guard-shell'
+  gem 'rubocop'
 end
 
 group :development do
-  gem 'rake', '~> 10.0'
-  gem 'rake-notes'
-  gem 'colorize'
   gem 'awesome_print'
+  gem 'colorize'
   gem 'pry'
   gem 'pry-byebug'
+  gem 'rake', '~> 10.0'
+  gem 'rake-notes'
 end
 
 group :test do
-  gem 'mocha'
-  gem 'webmock', '~> 1.18.0'
-  gem 'minitest-reporters', '~> 1.1.9'
+  gem 'coveralls'
   gem 'minitest-focus'
+  gem 'minitest-reporters', '~> 1.1.9'
+  gem 'mocha'
   gem 'pronto'
   gem 'pronto-flay', require: false
   gem 'pronto-rubocop', require: false
-  gem 'simplecov'
-  gem 'coveralls'
   gem 'rack', '~> 1.4.7'
-  gem 'vcr'
+  gem 'simplecov'
   gem 'timecop'
+  gem 'vcr'
+  gem 'webmock', '~> 1.18.0'
+end
+
+group :development, :test do
+  # Pronto
+  gem 'pronto'
```

Comment:
Bundler/DuplicatedGem: Gem `pronto` requirements already given on line 24 of the Gemfile.